### PR TITLE
Kill tabs dead! For good!

### PR DESCRIPTION
**Pull request by jpakkane**
I don't know who keeps adding those in but it stops now.

### CODE REVIEW

File: run_project_tests.py [hunk 1]
```diff
@@ -636,10 +636,29 @@ def check_file(fname):
...
+                      '.d',
+                      '.s',
+                      '.m',
+                      '.mm',
+                      '.asm',
+                      '.java',
+                      '.txt',
+                      '.py',
+                      '.swift',
+                    }
```
Comment (ignatenkobrain) [inline run_project_tests.py]: __[Flake8]__

[E124] closing bracket does not match visual indentation

File: run_project_tests.py [hunk 2]
```diff
@@ -636,10 +636,29 @@ def check_file(fname):
...
     for (root, _, files) in os.walk('.'):
-        for file in files:
-            if file.endswith('.py') or file.endswith('.build') or file == 'meson_options.txt':
```
Comment (textshell) [inline run_project_tests.py]: isn't this missing .build files?

File: manual tests/7 vala composite widgets/mywidget.vala [hunk 3]
```diff
@@ -1,41 +1,41 @@
-using Gtk;
+    using Gtk;
```
Comment (textshell) [inline manual tests/7 vala composite widgets/mywidget.vala]: This indent seems unintentional.

**Comment by jpakkane** [comment]
Updated.

File: run_project_tests.py [hunk 4]
```diff
@@ -636,10 +636,30 @@ def check_file(fname):
...
+                      '.swift',
+                      '.build',
+                      }
     for (root, _, files) in os.walk('.'):
-        for file in files:
-            if file.endswith('.py') or file.endswith('.build') or file == 'meson_options.txt':
-                fullname = os.path.join(root, file)
+        for fname in files:
+            if os.path.splitext(fname)[1].lower() in check_suffixes:
+                if os.path.split(fname)[1] == 'sitemap.txt':
```
Comment (QuLogic) [inline run_project_tests.py]: This is just `os.path.basename`.

**Pull request merged**